src/python/bot/tasks/fuzz_task.py

```diff
@@ -665,6 +666,20 @@ def pick_redzone():
   return utils.random_weighted_choice(redzone_list).size
 
 
+def pick_ubsan_disabled(job_type):
+  """Choose whether to disable UBSan in an ASan+UBSan build."""
+  # This is only applicable in an ASan build.
+  memory_tool_name = environment.get_memory_tool_name(job_type)
+  if memory_tool_name not in ['ASAN', 'HWASAN']:
+    return False
+
+  # Check if UBSan is enabled in this ASan build. If not, can't disable it.
+  if not environment.get_value('UBSAN'):
+    return False
+
+  return not utils.random_number(0, DEFAULT_CHOOSE_PROBABILITY)
```

Comment:
why only 10%? maybe 50?

Reply:
Starting slow. Also, i think for major targets we hurt fuzzing efficiency and lose ubsan bugs if we disable too many times.